spec/support/fetchable_behavior.rb

```diff
@@ -136,3 +173,41 @@
     end
   end
 end
+
+shared_examples_for 'a record with an indexable url' do
+  describe 'validations' do
+    context 'when the url extension is blacklisted' do
+      let(:movie_url) { "http://www.nps.gov/some.mov" }
+      let(:record) { described_class.new(valid_attributes.merge(url: movie_url)) }
+
+      it "is not valid" do
+        expect(record).not_to be_valid
+        expect(record.errors.full_messages.first).to match(/extension is not one we index/i)
+      end
+    end
+  end
+end
+
+shared_examples_for 'a record that belongs to a searchgov_domain' do
+  describe 'associations' do
+    it { is_expected.to belong_to(:searchgov_domain) }
```

Comment:
@ethanewing , I realized we are missing an association spec in `searchgov_domain_spec.rb` for the `has_many` relationship to `sitemaps`. I'll add that:
```ruby
it { is_expected.to have_many(:sitemaps).dependent(:destroy) }
```
Strict TDD would have helped, because you would have written the spec before adding the `has_many` relationship in the `SearchgovDomain` model.